app/src/openapi_server/impl/processes_api.py

```diff
@@ -0,0 +1,217 @@
+                # for input_id, input_value in execute_workflows.inputs.items():
+                #     input_description = next(
+                #         (input for input in process_description.inputs if input.id == input_id),
+                #         None,
+                #     )
+                #     if input_description is None:
+                #         raise HTTPException(
+                #             status_code=status.HTTP_400_BAD_REQUEST,
+                #             detail=f"Invalid input: {input_id}",
+                #         )
+                #     try:
+                #         #validate(instance=input_value.value, schema=input_description.schema_)
+                #         validated_inputs[input_id] = input_value.value
+                #     except ValidationError as e:
+                #         raise HTTPException(
+                #             status_code=status.HTTP_400_BAD_REQUEST,
+                #             detail=f"Invalid input for {input_id}: {e.message}",
+                #         )
+                #     validated_inputs[input_id] = input_value.value
+
+                job_id = str(uuid.uuid4())
+                logical_date = datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+                inputs_dict = {}
+                if execute_workflows.inputs:
+                    for key, value in execute_workflows.inputs.items():
+                        try:
+                            inputs_dict[key] = value.model_dump(exclude_unset=True)
+                        except Exception:
+                            inputs_dict[key] = value
+
+                data = {
+                    "dag_run_id": job_id,
+                    "logical_date": logical_date,
+                    "conf": inputs_dict,
+                }
+
+                airflow_response = requests.post(
+                    f"{self.settings.EMS_API_URL}/dags/{processId}/dagRuns",
+                    json=data,
+                    auth=self.ems_api_auth,
+                )
+                airflow_response.raise_for_status()
+
+                job = StatusInfo(
+                    processID=processId,
+                    type="process",
+                    jobID=job_id,
+                    status=StatusCode.ACCEPTED,
+                    created=datetime.now(),
+                    updated=datetime.now(),
+                )
+                crud.create_job(self.db, job.model_dump(by_alias=True))
+                if prefer == "respond-async":
+                    # Asynchronous execution
+                    return job
+                else:
+                    # Synchronous execution
+                    # Note: In a real-world scenario, you'd wait for the job to complete
+                    # and return the actual results. This is a simplified version.
+                    # TODO get result from job using the result endpoint and return it here.
+                    return Execute200Response(
```

Comment:
we're assuming we never call the sync mode here, correct? I'm fine with that- just need to be clear in documentation/conformance.